pandas/core/frame.py

```diff
@@ -5199,6 +5199,34 @@ def drop_duplicates(
         else:
             return result
 
+    def is_unique(
+        self, subset: Optional[Union[Hashable, Sequence[Hashable]]] = None
+    ) -> Series:
+        """
+        Return boolean Series denoting columns with unique values.
+
+        Parameters
+        ----------
+        subset : column label or sequence of labels, optional
+            Only consider certain columns for finding uniques. by default use columns.
+
+        Returns
+        -------
+        Series
+
+        See Also
+        --------
+        DataFrame.duplicated : Indicate duplicate rows.
+        """
+        if subset is not None:
+            subset = subset if is_list_like(subset) else [subset]
+            return self.loc[:, subset].is_unique()
+
+        if len(self.columns):
+            return self.apply(Series._is_unique)
```

Comment:
this should *just* work with
```return self.apply(lambda x: x.is_unique())```

Reply:
Seemed to be the the same, but I guess someone could have subclassed `_constructor_sliced`, which could make a difference.